2.0/config/log_config.py

```
@@ -0,0 +1,32 @@
+import logging
+import os
+import sys
+from logging.handlers import RotatingFileHandler
+
+def config_logging(service_name: str):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)